demo/screens/SellOrderExecuteScreen.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { connect } from "react-redux";
+import { removeSellOrder } from "../actions";
+import SellOrderExecute from "@presentational/SellOrderExecute";
+import PropTypes from "prop-types";
+import ContractsABIs from "@constants/ContractsABIs";
+const { manaABI, estateABI, marketplaceABI } = ContractsABIs;
+import ContractsAddresses from "@constants/ContractsAddresses";
+const { manaAddress, estateAddress, marketplaceAddress } = ContractsAddresses;
+import { Action } from "tasit-sdk";
+const { Contract } = Action;
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+export class SellOrderExecuteScreen extends React.Component {
+  // TODO: Switch to new DecentralandEstate() once the SDK includes that
+  estateContract = new Contract(estateAddress, estateABI);
+  marketplaceContract = new Contract(marketplaceAddress, marketplaceABI);
+  manaContract = new Contract(manaAddress, manaABI);
+
+  _onOrderExecution = sellOrder => {
+    const { navigation } = this.props;
+    navigation.navigate("ListSellOrdersScreen");
+    this._executeOrder(sellOrder);
```

Comment:
Do we want to do `_executeOrder` after the navigation? I think we probably do, but I want to make sure I understand the rationale. Thanks!

Reply:
Since blockchain operations are slow, I've thought that make the user go ahead and show
a loading/progress feedback (not implemented yet) would be better than keeping waiting. But I'm not sure. What do you think?

Reply:
I agree - that might be worth putting in the code as a note.

Could we also achieve that by dispatching an async action before navigating to the new screen?

Thanks!

Reply:
Yeah, I've done that, since `_executeOrder` is already async.
Missing move that to a `redux` action.

Reply:
Seems that [redux docs](https://redux.js.org/advanced/async-actions) recommends keep `actions` as plain functions and use middleware (thunk/saga) to handle with fetching data. What do you think?

Reply:
Yeah, I think that’s the way to go

Reply:
Can we extract this thread to #20 ?

Reply:
Yep! Feel free to resolve this once you do.